build/zip_bundle.gni

```diff
@@ -19,11 +19,25 @@ if (flutter_runtime_mode == "jit_release") {
 # The files variable is an array of scopes that specify a source file or
 # directory and a destination path in the archive to create.
 #
+# If this zip file is not intended to be codesigned as a Mac artifact,
+# set variable encode_entitlements to false.
+#
+# If encode_entitlements is set to true, this api will embed Mac
+# codesign entitlements information in the zip file at
```

Comment:
Maybe more explicitly stating here that it will embed a entitlement.txt file with the list of artifacts to get signed with entitlements ...